### PR TITLE
Ports tribal expansion

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
This does a bit of things
Adds a new weapon class: Bows
Adds a ton of bone weapons and tools for ash walkers to play with, We don't even have ash walkers, why am I porting this?

Ports https://github.com/vlggms/tegustation/pull/112 which in turn is a port of https://github.com/BeeStation/BeeStation-Hornet/pull/1802 which in turn is a port of 
Citadel-Station-13/Citadel-Station-13#11098
Citadel-Station-13/Citadel-Station-13#10993
Citadel-Station-13/Citadel-Station-13#11201
Citadel-Station-13/Citadel-Station-13#11401
Citadel-Station-13/Citadel-Station-13#11516
Citadel-Station-13/Citadel-Station-13#11172
Citadel-Station-13/Citadel-Station-13#11367

and some things cut out like the bounties and exports because... we don't have that
todo
- [x] modularize

## Why It's Good For The Game

Its only good for ashwalkers, which seem to be rare in the shiptest-verse, but this adds bows so i think thats good

## Changelog
:cl:
add: Why even write these changelogs? They aren't even shown in game. 
add: Added a Goliath shield and it takes less damage from fauna.
add: Added a new Watcher bola, slightly better than the standard bola.
add: Added a new one handed bone sword.
add: Added a new belt quiver to carry arrows.
add: Two new craftable bow types "pipe bow" and "bone bow".
add: 4 new arrow types wood, ash, bone and bronze.
add: New export prices for bows, arrows.
add: New bounty for a bow.
add: New variation of bone armor (heavy bone armor)
add: Cotton seeds in askwalker nest for bowcrafting.
add: QOL cultivator for ashwalkers added to nest.
tweak: blowgun no longer needs a firing pin and can be used by anyone
soundadd: New sounds for drawing the arrow and firing.
add: New weapon and ammo type for bows.
fix: Added a crafting recipe for the primal explorers belt.
add: A new material Silk which can be made from cloth on the loom. (needed in bowcrafting)
imageadd: Added Sprites for the new items and materials.
imageadd: New sprites for existing bone equipment.
refactor: Cloth is now a subtype of cotton and coded in descending order of creation.
crafting recipes are updated to reflect this.
Cotton>cloth>silk
refactor: Crafting refactor
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
Please link the original PR in your description.

**Comment by rye-rice** [comment]
after sitting on this for so long it should be good to go

File: code/modules/projectiles/guns/misc/bow.dm [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+/obj/item/gun/ballistic/bow/chamber_round(keep_bullet = FALSE, spin_cylinder, replace_new_round)
+	if(chambered || !magazine)
+		return
+	if(magazine.ammo_count())
+		chambered = magazine.get_round(TRUE)
+		chambered.forceMove(src)
+
+/obj/item/gun/ballistic/bow/attack_self(mob/user)
+	if(chambered)
+		to_chat(user, "<span class='notice'>You [drawn ? "release the tension on" : "draw the string on"] [src].</span>")
```
Comment (MarkSuckerberg) [inline code/modules/projectiles/guns/misc/bow.dm]:
```suggestion
		to_chat(user, "<span class='notice'>You [drawn ? "release" : "draw"] [src]'s string.</span>")
```

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
Looks good other than this, should be testmerged first, though

**Comment by triplezeta** [comment]
how

**Comment by rye-rice** [comment]
i will scream

**Comment by Pontenerd** [comment]
It's finally over

**Comment by rye-rice** [comment]
at long last

**Pull request merged**